### DISCUSSION TITLE
v7.4.0-preview.6 Release of PowerShell

### DISCUSSION THREAD

**Discussion by daxian-dbw** [PowerShell/PowerShell#20388] [Show and tell]
### General Cmdlet Updates and Fixes

- Set approved experimental features to stable for 7.4 release (#20362)
- Revert changes to continue using `BinaryFormatter` for `Out-GridView` (#20360)
- Remove the comment trigger from feedback provider (#20346)

### Tests

- Continued improvement to tests for release automation (#20259)
- Skip the test on x86 as `InstallDate` is not visible on `Wow64` (#20255)
- Harden some problematic release tests (#20254)

### Build and Packaging Improvements

<details>

<summary>

<p>Move to .NET 8.0.100-rc.1.23463.5</p>

</summary>

<ul>
<li>Update the regex for package name validation (Internal 27783, 27795)</li>
<li>Update ThirdPartyNotices.txt (Internal 27772)</li>
<li>Remove the <code>ref</code> folder before running compliance (#20375)</li>
<li>Updates RIDs used to generate component Inventory (#20372)</li>
<li>Bump <code>Microsoft.CodeAnalysis.CSharp</code> from 4.7.0 to 4.8.0-2.final (#20368)</li>
<li>Fix the release build by moving to the official .NET 8-rc.1 release build version (#20365)</li>
<li>Update the experimental feature JSON files (#20363)</li>
<li>Bump <code>XunitXml.TestLogger</code> from 3.1.11 to 3.1.17 (#20364)</li>
<li>Update <code>Microsoft.PowerShell.PSResourceGet</code> to 0.9.0-rc1 (#20361)</li>
<li>Update .NET SDK to version 8.0.100-rc.1.23455.8 (#20358)</li>
<li>Use <code>fxdependent-win-desktop</code> runtime for compliance runs (#20359)</li>
<li>Add mapping for mariner arm64 stable (#20348)</li>
<li>Bump <code>xunit.runner.visualstudio</code> from <code>2.5.0</code> to <code>2.5.1</code> (#20357)</li>
<li>Bump <code>JsonSchema.Net</code> from 5.2.1 to 5.2.5 (#20356)</li>
<li>Bump <code>Microsoft.NET.Test.Sdk</code> from 17.7.1 to 17.7.2 (#20355)</li>
<li>Bump Markdig.Signed from 0.32.0 to 0.33.0 (#20354)</li>
<li>Bump JsonSchema.Net from 5.1.3 to 5.2.1 (#20353)</li>
<li>Bump actions/checkout from 3 to 4 (#20352)</li>
<li>Bump <code>Microsoft.NET.Test.Sdk</code> from 17.7.0 to 17.7.1 (#20351)</li>
<li>Bump <code>Microsoft.CodeAnalysis.CSharp</code> from 4.7.0-2.final to 4.7.0 (#20350)</li>
<li>Release build: Change the names of the PATs (#20349)</li>
<li>Put the calls to <code>Set-AzDoProjectInfo</code> and Set-AzDoAuthToken` in the right order (#20347)</li>
<li>Bump Microsoft.Management.Infrastructure (continued) (#20262)</li>
<li>Bump <code>Microsoft.Management.Infrastructure</code> to 3.0.0-preview.2 (#20261)</li>
<li>Enable <code>vPack</code> provenance data (#20260)</li>
<li>Start using new packages.microsoft.com cli (#20258)</li>
<li>Add mariner arm64 to PMC release (#20257)</li>
<li>Fix typo <code>donet</code> to <code>dotnet</code> in build scripts and pipelines (#20256)</li>
</ul>

</details>

[7.4.0-preview.6]: https://github.com/PowerShell/PowerShell/compare/v7.4.0-preview.5...v7.4.0-preview.6

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - 515A88C7DB3BEEC37952A4D61B6F399BCB3242E17C1FBA79FCB181FE03F614F1
- powershell-7.4.0-preview.6-linux-arm32.tar.gz
  - 34F3856764C29E34345142A8B81564C6C3BB7E9E8CBF9CE907B567B9AC2DB2B2
- powershell-7.4.0-preview.6-linux-arm64.tar.gz
  - 4CC6C26D42DFF4235B9F3E4B2AA1FB528890C380AD9CEE8FACFF418C668F68EE
- powershell-7.4.0-preview.6-linux-linux-musl-x64.tar.gz
  - FAF4D357CCC3DFA0DDD181993A838AB51BCAC053AD1C356C0A8234FC2394CBB1
- powershell-7.4.0-preview.6-linux-x64.tar.gz
  - FD20B2B42BD61EC9E768D179EC35E2077A81C11DB375F92639355FD4E3E6313B
- powershell-7.4.0-preview.6-linux-x64-alpine-fxdependent.tar.gz
  - 58E98F76D94164E329373F505F4F98F675C6F4A1837DC8AD1FAEB4659B36BF5C
- powershell-7.4.0-preview.6-linux-x64-fxdependent.tar.gz
  - 714A2F6C419A3D770E4C5BBC26AEFED1F48D909F156E5805C97A24FA26DBE272
- powershell-7.4.0-preview.6-osx-arm64.pkg
  - 5DB594D8768668E4833890E5BF16C3B19761CD42C8BE9321AD631BB0603B74EA
- powershell-7.4.0-preview.6-osx-arm64.tar.gz
  - EDBE32089CBFB60892FC63AFD269E1726DBDA6AC607C99972388ED2BBEF4E86D
- powershell-7.4.0-preview.6-osx-x64.pkg
  - A3158336A91D87BDB0E2D735470D63389ADA0C66B1D709BBA23AF0476338D1E9
- powershell-7.4.0-preview.6-osx-x64.tar.gz
  - 2BCEEBC628740264383EE5774C22C3DF311D12D4DE5755BB345AB734381A91FE
- PowerShell-7.4.0-preview.6-win-arm64.zip
  - 1E4CF7642E563E874AEFA01C7A6A6E44C162DA16705CD9BDC3743788104C40FE
- PowerShell-7.4.0-preview.6-win-fxdependent.zip
  - 7EBB640E2EBA8D0173ED36A0FB5877CBC7762762F0BC53A377ACBF1A8B44E4DB
- PowerShell-7.4.0-preview.6-win-fxdependentWinDesktop.zip
  - 6A06AF2FC435F8E01D05B4E8BC782E148BCF6D608387F1D7C6E9C411281EAF64
- PowerShell-7.4.0-preview.6-win-x64.msi
  - C7046428033B6D36C2C6EF0B9893633DC3B8E12D8CD913AE086A563A51DA2B49
- PowerShell-7.4.0-preview.6-win-x64.zip
  - 4E52B774FEE4CA0B2C90A8D3ADFF45969178B10F735B94DED0E312BEB3301E6C
- PowerShell-7.4.0-preview.6-win-x86.msi
  - 72E368D4DB0F12EBB71178EA15EB73CF20DA70C58DC9ECF5EF3F682153EB31B9
- PowerShell-7.4.0-preview.6-win-x86.zip
  - 7AEB6D8CE288A45C537AB196B7AE0D4BC30F14D2C161A7ACFC75D87DC5D475EC
- powershell-preview_7.4.0-preview.6-1.deb_amd64.deb
  - 2EEAA3C725E80F22D6AA6147C4D0721BE2ED17340FCB706FEA47EBE9F0033C33
- powershell-preview-7.4.0_preview.6-1.cm.aarch64.rpm
  - CB4F2620BC37264B6E137639C190D2458DCE52DCF8FA2FF693373455F588E9E9
- powershell-preview-7.4.0_preview.6-1.cm.x86_64.rpm
  - 3596CC444436C2D995E1E252CD44D681DACD28DB2E6C3D192BAD4DD70279EDAF
- powershell-preview-7.4.0_preview.6-1.rh.x86_64.rpm
  - 6409088D11CC961FBA09B003E5C24118828C158903A4599B86E134D87BD32664

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.4.0-preview.6'>v7.4.0-preview.6 Release of PowerShell</a>.</em>

**Comment by jorgelevy** [comment]
Any ETA when this update will be available via Winget?

**Reply by ChadThomsen** [reply]
Why don't we make it part of the Winget repo/setup when it's released as part of the release process? This is not the first time I have seen this delay.

**Comment by merunaaa** [comment]
When will the msixbundle be added?
